apps/designer/app/designer/features/props-panel/use-props-logic.test.ts

```diff
@@ -350,7 +350,7 @@ describe("usePropsLogic", () => {
         name: "string",
         description: "string",
         location: "REMOTE",
-        createdAt: new Date("1995-12-17T03:24:00"),
+        createdAt: new Date("1995-12-17T03:24:00Z"),
```

Comment:
Will not help

Reply:
Why? Helped me, works on ci

Reply:
`delete result.current.userProps[0].asset.createdAt`

Reply:
Super, I thought `.toISOString()` issue.

Reply:
But why? iso string with z is absolute point in time, no?

Reply:
resolved forget about